docs/modules/surrealdb.md

```diff
@@ -0,0 +1,55 @@
+# SurrealDB
+
+Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+## Introduction
+
+The Testcontainers module for SurrealDB.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the SurrealDB module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/modules/surrealdb
+```
+
+## Usage example
+
+<!--codeinclude-->
+[Creating a SurrealDB container](../../modules/surrealdb/examples_test.go) inside_block:runSurrealDBContainer
+<!--/codeinclude-->
+
+## Module reference
+
+The SurrealDB module exposes one entrypoint function to create the SurrealDB container, and this function receives two parameters:
+
+```golang
+func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomizer) (*SurrealDBContainer, error)
+```
+
+- `context.Context`, the Go context.
+- `testcontainers.ContainerCustomizer`, a variadic argument for passing options.
+
+### Container Options
+
+When starting the SurrealDB container, you can pass options in a variadic way to configure it.
+
+#### Image
+
+If you need to set a different SurrealDB Docker image, you can use `testcontainers.WithImage` with a valid Docker image
+for SurrealDB. E.g. `testcontainers.WithImage("surrealdb/surrealdb:v1.1.1")`.
+
+{% include "../features/common_functional_options.md" %}
+
```

Comment:
Let's include here docs for the functional opts for the module: `WithStrict` and friends...